### ISSUE TITLE
[Feature Request] Cherrypick (copy) sticker size feature from nekogram

### ISSUE THREAD

**Issue by zarEclEC** [alissonlauffer/Customgram#2]
Here is the commit you need to cherrypick (and adapt to your client, I think) if you want to accept this feature request.
https://github.com/ArmeF97/Nekogram/commit/6054710c53fec1a9a2468e968d55b7707eb808a8

Basically it is the ability, client side, to resize stickers

Thanks

**Comment by alissonlauffer** [comment]
Yeah, it's a good feature. But I'll need to implement a customization menu first (or add it to chat settings?)

**Comment by HitaloM** [comment]
@alissonlauffer Add in the settings of the telegram itself, no need to create a new menu to have just it.

In the future when Customgram has more functions of its own, a separate menu will be more useful.

**Comment by alissonlauffer** [comment]
> @alissonlauffer Add in the settings of the telegram itself, no need to create a new menu to have just it.
> 
> In the future when Customgram has more functions of its own, a separate menu will be more useful.

I though the same.

**Comment by zarEclEC** [comment]
Do as you please, you are the owner. I think a separate menu is useful to not break things on merge from official client

**Comment by zarEclEC** [comment]
Maybe you are interested also in this
https://github.com/ArmeF97/Nekogram/commit/7919ec075a4a4ffdb2c43a9025aff3e6cebf79a0
